README.md

```diff
@@ -42,121 +83,100 @@ YOUR_DOCKER_HOST_IP      idptestbed
+    
+  - 2.4- edit `--form "metadata=` with your idp_metadata file -> `--form "metadata=@idp-metadata.xml" \`
+  
+  - 2.5- edit the `AUTH` field with your auth token (you can get this by connecting to an account, look at the network requests) -> `AUTH="1a2b3cxx88...55t"`
+   
+Note: You could probably use an API-KEY instead of the `AUTH` field but I'll let you figure that out yourselves.
+    
+3- In `body_first_name_last_name.json`:
+
+By default, you should set it like this:
+
+``` 
+{
+  "emailDomain": "test.com",
+  "idp": "https://idptestbed/idp/shibboleth",
+  "ppid": "urn:oid:0.9.2342.19200300.100.1.1",
+  "firstName": "urn:oid:2.5.4.4",
+  "lastName": "urn:oid:2.5.4.42"
+}
 ```
 
-The attributes sent are listed in the `attribute-filter.xml` file. The actual attribute names sent are themselves
-defined in the `attribute-resolver.xml` file. If you are not testing for specific attributes, you can use the following
-for the saml_attribute setup in the application's database (see below):
+If you changed the emailDomain earlier, change it here as well.
 
-- PPID: urn:oid:0.9.2342.19200300.100.1.1
-- FIRST_NAME: urn:oid:2.5.4.42
-- LAST_NAME: urn:oid:2.5.4.4
+The attributes sent (`ppid`, `firstName` and `lastName`), as defined above, are for a basic setup.
 
-So on your database, assuming your organization ID is `1` you would run the following:
+If you are testing for specific attributes, the attributes sent are listed in the `attribute-filter.xml` file. 
+The actual attribute names sent are themselves defined in the `attribute-resolver.xml` file. 
 
-```sql
-INSERT INTO saml_integration (domain, idp, organization_id) VALUES ('test.com','https://idptestbed/idp/shibboleth', 1);
+4- Rename `body_first_name_last_name.json` to `body.json` (or change the `--data @body.json \` field in `2_operations.sh`)
 
-INSERT INTO saml_attribute (type, name, saml_integration_id)
-SELECT 'PPID', 'urn:oid:0.9.2342.19200300.100.1.1', saml_integration.id
-FROM saml_integration
-WHERE saml_integration.idp = 'https://idptestbed/idp/shibboleth' AND saml_integration.domain = "test.com";
+5- Execute the query `1_before.sql` to make sure the saml_integration data is NULL.
 
-INSERT INTO saml_attribute (type, name, saml_integration_id)
-SELECT 'LAST_NAME', 'urn:oid:2.5.4.42', saml_integration.id
-FROM saml_integration
-WHERE saml_integration.idp = 'https://idptestbed/idp/shibboleth' AND saml_integration.domain = "test.com";
+6- Execute the script `2_operations.sh`.
 
-INSERT INTO saml_attribute (type, name, saml_integration_id)
-SELECT 'FIRST_NAME', 'urn:oid:2.5.4.4', saml_integration.id
-FROM saml_integration
-WHERE saml_integration.idp = 'https://idptestbed/idp/shibboleth' AND saml_integration.domain = "test.com";
-```
+7- Execute the query `3_after.sql` to make sure the script worked.
+
+8- Cherry-pick the commit `3457487470` (fix for local SSO)
```

Comment:
Add comment referencing the following [ticket](https://contractlive.atlassian.net/browse/CPM-6751)

Reply:
done